modules/cmpapi/package.json

```diff
@@ -88,5 +88,8 @@
     "type": "git",
     "url": "https://github.com/InteractiveAdvertisingBureau/iabtcf-es.git",
     "directory": "modules/cmpapi"
+  },
+  "dependencies": {
+    "@iabtcf/core": "^1.1.4"
```

Comment:
already listed in `peerDependencies`, which I _think_ is the right way to do this, so this should probably be removed.

Reply:
Yes that’s right! You don’t want to include the entire core library with every CMP API

Reply:
This still needs to be addressed.

Reply:
Removed